docs/decisions/0013-system-taxonomy-auto-tagging.rst

```diff
@@ -0,0 +1,30 @@
+13. System-defined automatic tagging
+=====================================
+
+Context
+--------
+
+One main characteristic of the System-defined is the automatic content tagging. 
+It is necessary to implement this functionality when the associated content is created or edited.
+
+Decision
+---------
+
+Use `openedx-filters`_ to call the auto tagging function after content creation/edition.
+After create the ``Content-side`` class, `register a pipeline`_ with the respective filter.
+Some filters must to be created, like ``CourseCreation``, ``LibraryCreation``, etc.
+
+All this logic will be live on ``openedx.features.tagging``.
```

Comment:
I think it's better for the new filters themselves to live outside of `openedx.features.tagging`?

The filter logic itself will still live in `openedx-filters`, and the events that trigger the filters should live as close to the content creation as possible. The `openedx.features.tagging` code should simply register to receive these events and act on them -- as may other `openedx` libraries that care about content creation.

Reply:
@pomegranited Yes, I updated the doc to be more clear about it :+1: